ui/src/components/Tasks/Characterisation.jsx

```diff
@@ -139,6 +142,9 @@ class Characterisation extends React.Component {
                 label="Exposure time (s)"
               />
               <InputField
+                disabled={
+                  this.props.beamline.hardwareObjects.resolution.readonly
+                }
```

Comment:
resolution depends on the detector distance, not just the energy. I will be very surprised if it is ever read only